webnn/resources/utils.js

```diff
@@ -13,6 +13,47 @@ const TypedArrayDict = {
   float32: Float32Array,
 };
 
+/**
+ * Get ULP tolerances of gemm operation.
+ * @param {Array} shapeA [number, number]
+ * @param {Object} options
+ * @param {Object} [options.c]
+ * @param {Number} [options.alpha]
+ * @param {Number} [options.beta]
+ * @param {Boolean} [options.aTranspose]
+ * @param {Boolean} [options.bTranspose]
+ * @returns {Number}
+ */
+function gemmULPTolerances(shapeA, options) {
```

Comment:
@fdwr Please take a review of this implementation of gemm ULP tolerances, thanks.

Reply:
A comment somewhere would be helpful. e.g.

// GEMM : alpha * (A x B) + beta * C
// An upper bound for the worst serial ordering is bounded by
// the number of lossy operations, where matrix multiplication
// is a dot product (mul and add times the number of elements)
// plus bias operations.